### DISCUSSION TITLE
Sampling variable at a plane in 3D

### DISCUSSION THREAD

**Discussion by aaelmeli** [idaholab/moose#20664] [Q&A General] [answered]
Hi

I need to sample the solution of a problem in a plane in 3D. I was thinking of something like `LineValueSampler` but I did not find something similar. So, I have tried to define new nodesets using `BoundingBoxNodeSetGenerator`, and then try to output the variable on those nodes using `VectorPostprocessors` but I could not. 
Any suggestions?

**Comment by GiudGiud** [comment, marked as answer]
Hello

So you'd want something like PlaneValueSampler? 
With a regular grid overlaid over the domain?

Guillaume

**Reply by aaelmeli** [reply]
> Hello
> 
> So you'd want something like PlaneValueSampler? With a regular grid overlaid over the domain?
> 
> Guillaume

Yes, exactly.

**Reply by aaelmeli** [reply]
> Hello
> 
> So you'd want something like PlaneValueSampler? With a regular grid overlaid over the domain?
> 
> Guillaume

I figured it out. the solution that I have suggested initially worked out, i.e `BoundingBoxNodeSetGenerator`. The problem was that I was using `GeneratedMesh `instead of `GeneratedMeshGenerator`  in the Mesh block.

**Reply by GiudGiud** [reply]
I'm glad. If we ever need a solution for less regular grids we can add a PlaneValueSampler that's not a problem
